ansible/roles/uan_packages/defaults/main.yml

```diff
@@ -28,21 +28,30 @@
 
 # See the role README.md file for information about this variable.
 ---
+uan_disable_gpg_check: no
+
 uan_sles15_repositories_add:
   - name: "uan-2.3-sle-15sp3"
     description: "UAN SUSE Linux Enterprise 15 SP3 Packages"
     repo: "https://packages.local/repository/uan-2.3-sle-15sp3"
     priority: 2
+    disable_gpg_check: "{{ uan_disable_gpg_check }}"
 
 uan_sles15_packages_remove:
   - cray-cos-release
 
 uan_sles15_packages_add:
-  - cray-diags-fabric
```

Comment:
This does the job, but I think a future enhancement might create a dictionary list of the packages with the same setting for the gpg check. Previously we could zypper install all of the rpms in one ansible task. With this change, it's a zypper install task for each package.